### PR TITLE
[Doc Improvement][Update rsc for team members][3208223]

### PR DESCRIPTION
**Pull request by v-ypalikila**
Updated the table to state members of the team to install and uninstall any kind of app inside the team as per [channel post](https://teams.microsoft.com/l/message/19:814416796bf34cfb82c9a15b0c63becf@thread.skype/1689763747577?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=32e3b156-66b2-4135-9aeb-73295a35a55b&parentMessageId=1689763747577&teamName=Teams%20Platform&channelName=Announcements&createdTime=1689763747577).

### CODE REVIEW

**Review by v-habibaumme — APPROVED** [review]

**Review by AkJo — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/64c63246f3de4eafcd4410188e8c6f97ead499cf/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 99 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3214abb3-817f-4c6c-ab90-92da1db47271) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/63815948ab00a06e1768122290361bc5a6027ad8/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 99 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/27ba500c-0544-4965-8f4a-e5ed881eb9ee) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8e9309294aceb7449b8bbdcc42910a52d57927fb/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 99 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dc006f04-b3d6-4a40-9667-f5d4b07a9083) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b965131018e8b0e8c14f7ae0897079991fbb4183/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 99 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8284ff3f-1403-4d90-ba3c-da8c0f8dd505) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1beb943c08d207c294ef46a8857e05e95b64b1f2/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 99 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/267af2b8-5ab7-41f2-9c90-05b336d25ce4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/rsc/resource-specific-consent.md [hunk 1]
```diff
@@ -49,13 +49,27 @@ Use RSC permissions to determine the data access methods for your app. A user's
...
+
+Team owners can perform the following steps to allow members to add or remove apps with RSC permissions in a team:
+
+1. In Teams, go to a team, and select **More options (...)** > **Manage team**.
+
+1. Select **Settings** and expand **Member permissions**.
+
+1. Select **Allow members to add and remove apps**.
+
+Team owners can also allow members to upload and install custom apps. For more information, see [allow members to upload custom apps](/microsoftteams/teams-custom-app-policies-and-settings?branch=main).
```
Comment (AkJo) [inline msteams-platform/graph-api/rsc/resource-specific-consent.md:64]: This section does not make sense here.
1. This RSC overview is not talking about how apps works specifically inside a team. 
2. These instructions are not specific to RSC. There are general instructions about managing all types of apps within any team.

Please remove this section altogether, or find a new place for it (perhaps a See Also that lists out how to control who can install apps in which scopes).

Please work with Ricky for suggestions.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9d641d2ec1caf6a375693795b943cea4d961c80a/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 99 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/39ae5023-9931-425b-82c7-ccf69513090b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/rsc/resource-specific-consent.md (hunk 1 above)
Comment (v-ypalikila) [reply msteams-platform/graph-api/rsc/resource-specific-consent.md:64]: Removed the section and updated the  link in See also.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a71bdacb9f0db06a9755e18d1d1dc0a561796eb1/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 99 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dec4f39b-dd7e-4565-a5fb-aeaec1bc56d2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e6327e72c8c3ca72f2be576f3280cf7c0c962f1f/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 99 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/46c09954-7258-44ea-b2bd-a418eb91c310) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/769a4df0a6c3dbef893b3609f5149bc18c4cbecf/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 99 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/33b5adda-950c-469e-bdbe-56340f31d347) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**